### PR TITLE
Add url to contributor name

### PR DESCRIPTION
**Pull request by vkatsikaros**
I noticed I hadn't a url against my name, now adding.

***
<a href="https://pr-preview.s3.amazonaws.com/vkatsikaros/webdriver/pull/1846.html" title="Last updated on Oct 22, 2024, 2:24 PM UTC (14364fe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver/1846/db06684...vkatsikaros:14364fe.html" title="Last updated on Oct 22, 2024, 2:24 PM UTC (14364fe)">Diff</a>

### CODE REVIEW

**Comment by vkatsikaros** [comment]
I linked my (new) W3C and github accounts together, and clicked on "revalidate" at https://labs.w3.org/repo-manager/pr/id/w3c/webdriver/1846 but it seems the "user affiliation" still can't be determined.

**Comment by tidoust** [comment]
> I linked my (new) W3C and github accounts together, and clicked on "revalidate" at https://labs.w3.org/repo-manager/pr/id/w3c/webdriver/1846 but it seems the "user affiliation" still can't be determined.

The "revalidate" button bugs from time to time in my experience. In any case, the IPR bot would have continued to complain because you're not a Working Group participant. I marked the pull request as non-substantive.

File: index.html [hunk 1]
```diff
@@ -11512,7 +11512,7 @@ <h2>Acknowledgements</h2>
 <!-- Simon Stewart --> <a href=http://www.rocketpoweredjetpants.com/>Simon Stewart</a>,
 <!-- Sri Harsha --> Sri Harsha,
 <!-- Titus Fortner --> Titus Fortner,
-<!-- Vangelis Katsikaros --> and Vangelis Katsikaros.
+<!-- Vangelis Katsikaros --> and <a href=http://www.vkatsikaros.com/>Vangelis Katsikaros</a>.
```
Comment (whimboo) [inline index.html:11515]: While quickly discussing this addition we tend more to remove the URLs all together given that it we cannot guarantee that the page is still owned by that person. Instead we could add the GitHub handle.

```suggestion
<!-- Vangelis Katsikaros --> and <a href=https://github.com/vkatsikaros>Vangelis Katsikaros</a>.
```

But we should most likely do this on a different PR for everyone.

**Review by whimboo — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (vkatsikaros) [reply index.html:11515]: @whimboo makes sense, we expect the spec to be around for a long time (decades even?). Sorry for the noise!
